examples/ltsf/test_ltsf.py

```diff
@@ -0,0 +1,8 @@
+from sktime.forecasting.ltsf import LTSFLinearForecaster
+from sktime.utils.estimator_checks import check_estimator
+
+res = check_estimator(
```

Comment:
this should not be done like this - the test will be carried out automatically if the estimator is collected.

Reply:
test skips for the suite are configured globally, in `tests._config`